aptos-move/e2e-move-tests/src/aggregator_v2.rs

```diff
@@ -26,20 +40,31 @@ pub fn initialize(path: PathBuf) -> (MoveHarness, Account) {
     (harness, account)
 }
 
+fn create_2_arg(
+    harness: &mut MoveHarness,
+    account: &Account,
+    name: &str,
+    index: u64,
+    second: u128,
+) -> SignedTransaction {
+    harness.create_entry_function(account, str::parse(name).unwrap(), vec![], vec![
```

Comment:
Generally speaking, should we leave the `TypeTag` as empty for all the tests? In the native functions, we explicitly read the type tag and behave according to it.